### ISSUE TITLE
Number of Multi-device support.

### ISSUE THREAD

**Issue by Gazolini** [bambulab/BambuStudio#4603]
Hi.

Why Multi-device feature support no more than six devices? Can you add more devices, for example more than ten devices.

**Comment by luder888** [comment]
Agree. They need to support more than 6 devices and also make the sorted column PERSISTENT!
